[2023-06-15T05:50:17.589+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T05:50:04+00:00 [queued]>
[2023-06-15T05:50:17.594+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T05:50:04+00:00 [queued]>
[2023-06-15T05:50:17.595+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 4
[2023-06-15T05:50:17.602+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): pyspark_train_classifier_model> on 2023-06-15 05:50:04+00:00
[2023-06-15T05:50:17.607+0000] {standard_task_runner.py:57} INFO - Started process 1077 to run task
[2023-06-15T05:50:17.611+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'deploy_prediction_model', 'pyspark_train_classifier_model', 'manual__2023-06-15T05:50:04+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/deploy.py', '--cfg-path', '/tmp/tmpkfwlapt9']
[2023-06-15T05:50:17.613+0000] {standard_task_runner.py:85} INFO - Job 31: Subtask pyspark_train_classifier_model
[2023-06-15T05:50:17.662+0000] {task_command.py:410} INFO - Running <TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T05:50:04+00:00 [running]> on host 6426fae5ea9e
[2023-06-15T05:50:17.727+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deploy_prediction_model' AIRFLOW_CTX_TASK_ID='pyspark_train_classifier_model' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T05:50:04+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T05:50:04+00:00'
[2023-06-15T05:50:17.730+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-15T05:50:17.730+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\nspark-submit --class org.fiware.cosmos.orion.spark.connector.prediction.PredictionJob    --conf spark.driver.extraJavaOptions=-Dlog4jspark.root.logger=WARN,console   --master spark://spark-master:7077   //prediction-job/target/orion.spark.connector.prediction-1.0.1.jar   /']
[2023-06-15T05:50:17.740+0000] {subprocess.py:86} INFO - Output:
[2023-06-15T05:50:17.742+0000] {subprocess.py:93} INFO - /bin/bash: line 2: spark-submit: command not found
[2023-06-15T05:50:17.742+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2023-06-15T05:50:17.754+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2023-06-15T05:50:17.757+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=deploy_prediction_model, task_id=pyspark_train_classifier_model, execution_date=20230615T055004, start_date=20230615T055017, end_date=20230615T055017
[2023-06-15T05:50:17.765+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 31 for task pyspark_train_classifier_model (Bash command failed. The command returned a non-zero exit code 127.; 1077)
[2023-06-15T05:50:17.782+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-15T05:50:17.796+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
